pkg/operator/ceph/cluster/watcher.go

```diff
@@ -219,7 +219,10 @@ func (c *clientCluster) fenceNode(ctx context.Context, node *corev1.Node, cluste
 		return nil
 	}
 
-	clusterInfo := cephclient.AdminClusterInfo(ctx, cluster.Namespace, cluster.Name)
+	clusterInfo, _, _, err := opcontroller.LoadClusterInfo(c.context, ctx, cluster.Namespace, &cluster.Spec)
```

Comment:
this will also apply to the converged mode, I hope it is fine

And you may do end to end testing for this

Reply:
I have done the testing

Reply:
great it should work,
And do you see now rbd cmd running with the healthchecker

Reply:
And wanted to understand what does this mean https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh#L29